plugins/modules/system/pids.py

```diff
@@ -78,11 +78,21 @@ def compare_lower(a, b):
 def get_pid(name):
     pids = []
 
-    for proc in psutil.process_iter(attrs=['name', 'cmdline']):
-        if compare_lower(proc.info['name'], name) or \
-                proc.info['cmdline'] and compare_lower(proc.info['cmdline'][0], name):
-            pids.append(proc.pid)
-
+    try:
+        for proc in psutil.process_iter(attrs=['name', 'cmdline']):
+            if compare_lower(proc.info['name'], name) or \
+                    proc.info['cmdline'] and compare_lower(proc.info['cmdline'][0], name):
+                pids.append(proc.pid)
+    except TypeError:  # EL6, EL7: process_iter() takes no arguments (1 given)                        |
```

Comment:
I don't think the trailing spaces/pipe are needed:
```suggestion
    except TypeError:  # EL6, EL7: process_iter() takes no arguments (1 given)
```

Reply:
Great catch.